core/arch/arm/mm/core_mmu_v7.c

```diff
@@ -69,9 +69,16 @@
 /* Normal memory, Outer Write-Back Write-Allocate Cacheable */
 #define TEE_MMU_TTB_RNG_WBWA    (1 << 3)
 
+/* Normal memory, Outer Write-Back no Write-Allocate Cacheable */
+#define TEE_MMU_TTB_RNG_WB      (3 << 3)
+
+#ifndef CFG_NO_SMP
```

Comment:
Linux with LPAE uses different attributes which seem to match OP-TEE, I have not done extensive analysis, but testing on our LPAE platforms seems to indicate this, I do not have an ARMv8 platform to test but it should be the same case as for LPAE.